libs/libc/stdlib/lib__Exit.c

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * libs/libc/stdlib/lib_Exit.c
+ * libs/libc/stdlib/lib__Exit.c
```

Comment:
let's create a new PR to merge this patch quickly?

Reply:
Of course